assets/css/_garage_filter.scss

```diff
@@ -1,37 +1,74 @@
 .m-garage-filter {
   @include font-small;
-  color: #6e717d;
+  font-weight: 400;
+  color: #525560;
   background: $white;
-  border: 1px solid #9fa2ac;
-  border-radius: 0.25rem;
+  border: 1px solid #b3b5bd;
```

Comment:
**suggestion (non-blocking):** This value is used in several places. It might be worthwhile turning it into a variable.